src/server/app/models/meeting.js

```diff
@@ -1,14 +1,21 @@
 module.exports = function (sequelize, DataTypes) {
 
   var Meeting = sequelize.define('Meeting', {
-    title: DataTypes.STRING,
-    closeoutTime: DataTypes.DATE,
-    generalLocation: DataTypes.STRING,
-    duration: DataTypes.INTEGER
+    title: { type: DataTypes.STRING, allowNull: false },
+    closeoutDate: { type: DataTypes.DATE, allowNull: false},
```

Comment:
It says [here](http://docs.sequelizejs.com/variable/index.html#static-variable-DataTypes) that DataTypes.DATE is a datetime, so I don't think we need the DataTypes.TIME as well. I'd rename it back to `closeoutTime`.